src/studio/src/designer/frontend/dashboard/features/cloneService/cloneServices.tsx

```diff
@@ -129,7 +122,14 @@ export class CloneServiceComponent extends React.Component<ICloneServiceComponen
     });
 
     return returnService.length === 1 ? returnService[0] : null;
-  }
+  };
+
+  public redirectToCode = () => {
+    const repoInfo = this.getCurrentRepositoryInfo();
+    window.location.assign(`/repos/${repoInfo.full_name}`);
+  };
+
+  public componentMounted = false;
```

Comment:
Does it make sense for this state to be public?

Reply:
I don't know, I basically just renamed and moved it to a different line for the linter to shut up :)

Reply:
Okay, I see 😅 Probably fine to leave it as-is. We'll probably revisit this component later and rewrite to hooks anyway (at least I hope we will...)